libs/execution_context/ebpf_native.c

```diff
@@ -222,7 +204,7 @@ _ebpf_native_clean_up_maps(
             ebpf_free(map->pin_path.value);
 #pragma warning(pop)
         }
-        if (close_handles && map->handle != ebpf_handle_invalid) {
+        if (map->handle != ebpf_handle_invalid) {
```

Comment:
Should the above unpin/free be inside the check for `ebpf_handle_invalid` or is there a case where the handle could be invalid and the map pinned?

Reply:
@saxena-anurag What do you think? I did it this way to minimize churn.